api/src/certification/session-management/domain/services/session-publication-service.js

```diff
@@ -130,10 +135,14 @@ async function _manageCleaEmails({ session, certificationCenterRepository, sessi
 
 /**
  * @param {Object} params
+ * @param {Array<number>} params.startedCertificationCoursesUserIds
  * @param {MailService} params.mailService
  */
-async function _managerPrescriberEmails({ session, mailService }) {
-  const recipientEmails = _distinctCandidatesResultRecipientEmails(session.certificationCandidates);
+async function _managerPrescriberEmails({ session, startedCertificationCoursesUserIds, mailService }) {
```

Comment:
```suggestion
async function _managePrescriberEmails({ session, startedCertificationCoursesUserIds, mailService }) {
```

Quitte a modifier la ligne proposition de fix la typo avant